README.md

```diff
@@ -35,82 +35,146 @@ Connect joy via nano USB receiver to NUC and make sure it is in **DirectInput Mo
+
+- `~max_linear_vel` *(float, default: 0.2)*
+- `~min_linear_vel` *(float, default: 0.05)*
+- `~max_angular_vel` *(float, default: 0.5)*
+- `~min_angular_vel` *(float, default: 0.05)*
+- `~gripper_closed_position` *(float, default: 0.8)*
+
+---
+
+## Controlling Kinova manipulator and mobile robot
+
+By default, after a successful launch manipulator will be controlled by the gamepad. To start driving the robot and disable the manipulator press the **LB button** (disable movement). To switch back to using the manipulator press the **A button** (enable movement).
+
+### Manipulator button mapping
+
+|      Button      |           Function           |
+| :--------------: | :--------------------------: |
+|   `LEFT STICK`   | manipulator steering - `X Y` |
+|  `RIGHT STICK`   |  manipulator steering - `Z`  |
+|  `UP CROSS KEY`  |        increase speed        |
+| `DOWN CROSS KEY` |        decrease speed        |
+|       `A`        |       enable movement        |
+|       `B`        |        emergency stop        |
+|       `X`        |         linear mode          |
+|       `Y`        |         angular mode         |
+|       `LB`       |       disable movement       |
+|       `LT`       |        close gripper         |
+|       `RT`       |         open gripper         |
+
+To move the manipulator use sticks. By default, cartesian *linear/angular* movement in `X` and `Y` is held by the left stick and *linear/angular* `Z` movement is controlled with the right stick.
+
+### Robot button mapping
+
+|    Button     |            Function             |
+| :-----------: | :-----------------------------: |
+| `LEFT STICK`  | velocity steering - angular `Z` |
+| `RIGHT STICK` | velocity steering - linear `X`  |
+|     `LB`      |         enable driving          |
+|     `RB`      |        slow driving mode        |
+|     `RT`      |        fast driving mode        |
 
-## Running demo on Panther robot
+If neither `RB` nor `RT` is pressed, the robot operates in *regular* driving mode.
+
+---
+
+## Running on Panther robot
 
 1. Access NUC via ssh
 
-```bash
-ssh husarion@10.15.20.3
-```
+    ```bash
+    ssh husarion@10.15.20.3
+    ```
 
-default password is `husarion`
+    The default password is `husarion`
 
 2. Clone this repo
 
-```bash
-cd ~/husarion_ws/src/
-git clone https://github.com/husarion/kortex_joystick_demo.git
-```
+    ```bash
+    cd ~/husarion_ws/src/
+    git clone https://github.com/husarion/kortex_joystick_demo.git
+    ```
 
-3. Setup gamepad according to [Setup joy](#setup-joy)
+3. Setup a gamepad according to [Setup joy](#setup-joy)
 
 4. Setup virtual desktop
 
-```bash
-cd kortex_joystick_demo/
-. ./kortex_joystick_demo/scripts/setup_virtual_desktop.sh
-```
+    ```bash
+    cd kortex_joystick_demo/
+    . ./kortex_joystick_demo/scripts/setup_virtual_desktop.sh
+    ```
 
 5. Launch demo
 
-```bash
+    ```bash
+    ```
 
-docker compose -f compose.panther-kinova.yaml up
-```
+<!-- More info needed here TBD -->
```

Comment:
Is this intentional?

Reply:
Yep, this package requires more changes which won't be covered in this task. 
(Probably when testing will be possible again and code review will be provided, then those lacking parts will be covered)

Reply:
:+1: